pkg/cbfs/file.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2018-2021 the LinuxBoot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cbfs
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/linuxboot/fiano/pkg/compression"
+)
+
+var CbfsHeaderMagicNotFound = errors.New("CBFS header magic doesn't match")
+
+func (f *File) MarshalJSON() ([]byte, error) {
+	return json.Marshal(mFile{
+		Name:  f.Name,
+		Start: f.RecordStart,
+		Size:  f.FileHeader.Size,
+		Type:  f.FileHeader.Type.String(),
```

Comment:
Let's add `Compression` here (and for `mFile`) as well, then it's consistent with the string representation. :)